docs/script.js

```diff
@@ -0,0 +1,58 @@
+window.addEventListener("DOMContentLoaded", (event) => {
+  console.log("DOM fully loaded and parsed");
+
+  let idtAddBtn = document.getElementById('idt-add-btn');
+
+  let idtMain = document.getElementById('idt-QAs');
+
+  idtAddBtn.addEventListener("click", (event) => {
+    idtMain.innerHTML += `<br>
+    <label for="question">Q:
```

Comment:
Validation error: "The value of the for attribute of the label element must be the ID of a non-hidden form control."